src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2013,51 +2019,78 @@
       "description": "The analysis tool that was run.",
       "additionalProperties": false,
       "type": "object",
+      "properties": {
+
+        "language": {
+          "description": "The tool language (expressed as an ISO 649 two-letter lowercase culture code) and region (expressed as an ISO 3166 two-letter uppercase subculture code associated with a country or region).",
+          "type": "string",
+          "default": "en-US"
+        },
+
+        "driver": {
+          "description": "The analysis tool that was run.",
+          "$ref": "#/definitions/toolComponent"
+        },
+
+        "extensions": {
+          "description": "Tool extensions that contributed to or reconfigured the analysis tool run.",
+          "type": "array",
+          "minItems": 0,
+          "uniqueItems": true,
+          "default": [],
+          "items": {
+            "$ref": "#/definitions/toolComponent"
+          }
+        },
+
+
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the tool.",
+          "$ref": "#/definitions/propertyBag"
+        }
+      },
+
+      "required": [ "driver" ]
+    },
+
+    "toolComponent": {
+      "description": "A component, such as a plug-in or the default driver, of the analysis tool that was run.",
+      "additionalProperties": false,
+      "type": "object",
       "properties": {
 
         "name": {
-          "description": "The name of the tool.",
+          "description": "The name of the tool driver or extension.",
```

Comment:
>driver or extension [](start = 47, length = 19)

How about "component" since that's what you use in the rest of the properties below.